server/base/src/main/java/org/apache/accumulo/server/security/handler/ZKSecurityTool.java

```diff
@@ -114,16 +120,27 @@ public static byte[] createPass(byte[] password) throws AccumuloException {
     return cryptHash.getBytes(UTF_8);
   }
 
+  private static final Map<List<Byte>,String> CHECKED_CRYPT_PASSWORDS =
```

Comment:
Hadoop's Text or Accumulo's ByteSequence would probably be more efficient for the key than List<Byte>.

```suggestion
  private static final Map<ByteSequence,String> CHECKED_CRYPT_PASSWORDS =
```

Also thinking a guava cache w/ extremely short TTL (like 1 min) would be better than fixed size lrumap.  An extremely short duration should still amortize the cost nicely.